doc/howto/network_bridge_firewalld.md

```diff
@@ -1,52 +1,109 @@
-```
-lxc network set lxdbr0 ipv4.nat false
-lxc network set lxdbr0 ipv6.nat false
-lxc network set lxdbr0 ipv6.firewall false
-lxc network set lxdbr0 ipv4.firewall false
-```
+### UFW: Add rules for the bridge
 
-Finally, to enable iptables firewalld's rules for LXD usecase (in this example, we suppose the bridge interface is `lxdbr0` and the associated IP range is `10.0.0.0/24`:
-```
-firewall-cmd --permanent --direct --add-rule ipv4 filter INPUT 0 -i lxdbr0 -s 10.0.0.0/24 -m comment --comment "generated by firewalld for LXD" -j ACCEPT
-firewall-cmd --permanent --direct --add-rule ipv4 filter OUTPUT 0 -o lxdbr0 -d 10.0.0.0/24 -m comment --comment "generated by firewalld for LXD" -j ACCEPT
-firewall-cmd --permanent --direct --add-rule ipv4 filter FORWARD 0 -i lxdbr0 -s 10.0.0.0/24 -m comment --comment "generated by firewalld for LXD" -j ACCEPT
-firewall-cmd --permanent --direct --add-rule ipv4 nat POSTROUTING 0 -s 10.0.0.0/24 ! -d 10.0.0.0/24 -m comment --comment "generated by firewalld for LXD" -j MASQUERADE
-firewall-cmd --reload
-```
-To check the rules are taken into account by firewalld:
-```
-firewall-cmd --direct --get-all-rules
+If UFW has a rule to drop all unrecognised traffic, it blocks the traffic from and to the LXD bridge.
+In this case, you must add rules to allow traffic from and to the bridge.
+
+To do so, run the following commands:
+
+    sudo ufw allow in on <network_bridge>
+    sudo ufw route allow in on <network_bridge>
+
+For example:
+
+    sudo ufw allow in on lxdbr0
+    sudo ufw route allow in on lxdbr0
+
+## Prevent rule conflicts
+
+Linux firewalls are based on netfilter.
+LXD uses the same subsystem, which can lead to conflicts if the defined rules overlap.
+For example, the firewall might erase LXD rules if it is started after the LXD daemon, which would prevent the LXD container to do any outbound internet access.
+
+In this case, you should directly configure the rules defined by LXD in the firewall, and disable them in LXD.
+
+### xtables vs. nftables
+
+There are different userspace commands to add rules to netfilter: xtables (iptables for IPv4 and ip6tables for IPv6) and nftables.
+
+Xtables provides an ordered list of rules, which might cause issues if multiple systems add and remove entries from the list.
+Nftables adds the ability to separate rules into namespaces, which allows multiple systems to add rules that won't affect each other.
+
+If your system supports and uses nftables, LXD detects this and switches to nftables mode.
+In this mode, LXD adds its rules into the nftables, using its own nftables namespace.
+As a result, there are usually no conflicts with other applications.
+
+If your system uses xtables, on the other hand, you might see rule conflicts.
+In this case, follow the instructions below to configure LXD's rules directly in xtables instead of having LXD add them automatically.
+
+### Disable LXD's xtables rules
```

Comment:
> Disable LXD's firewall rules

Reply:
As this section applies to both xtables and nftables (which ever firewalld or ufw are using).